test/ConductorSharp.Engine.Tests/Samples/Workflows/StringInterpolation.cs

```diff
@@ -18,15 +20,16 @@ public class StringInterpolationOutput : WorkflowOutput
     }
 
     [OriginalName("TEST_StringInterpolation")]
-    public class StringInterpolation : Workflow<StringInterpolationInput, StringInterpolationOutput>
+    public class StringInterpolation : Workflow<StringInterpolation, StringInterpolationInput, StringInterpolationOutput>
```

Comment:
I would make separate test case where all features of new builder are used

Reply:
I would actually like to move to a major version change and use only the configurable builder before publishing the new release. The code and usage could be a bit confusing with these versions existing together.

Reply:
Even if we move to a newer version we can not simply remove the old builder because there are existing projects using it, hence we can not remove test cases. We can only deprecate it and iteratively remove replace the test cases

Reply:
I'm afraid these duplicated builders (configurable and non configurable) would cause naming confusion and lead us to invent complicated names for namespaces and classes. I would rather say we are moving to a new major version and have anybody who follows update their code to use configurable builders.

Reply:
Merging this as we agreed outside of this chat to move on with the major version changes :)